modules/ROOT/examples/live-demos/inline/index.html

```diff
@@ -1,6 +1,6 @@
 <div class="demo-inline">
   <div class="container">
-    <p><img src="https://www.tiny.cloud/images/email-banner-2.png" style="width: 100%;"/></p>
+    <p><img src="../_images/tiny-banner.png" style="width: 100%;"/></p>
```

Comment:
This shouldn't be using a hardcoded image URL. It should be using the AsciiDoc `imagedir` variable instead, as the location can change based on the Antora configuration.
```suggestion
    <p><img src="{{imagesdir}}/tiny-banner.png" style="width: 100%;"/></p>
```